.travis.yml

```diff
@@ -26,12 +32,15 @@ before_install:
   - ./runisort
 
 install:
+  - pip install $DJANGO
   - python setup.py -q develop
-  - if [[ $TRAVIS_PYTHON_VERSION == "2.7" ]]; then pip install git+git://github.com/tinio/pysqlite.git@extension-enabled#egg=pysqlite; fi
 
 script:
   - coverage run --source=openwisp_firmware_upgrader runtests.py
-  - if [[ $TRAVIS_PYTHON_VERSION == 3.4 ]]; then ./tests/manage.py makemigrations --dry-run | grep "No changes detected"; fi
```

Comment:
Do I Need To Keep This ?

Reply:
no because it's done by `openwisp-utils-qa-checks`

Reply:
Okay :smiley: